src/microbe_stage/MicrobeStage.cs

```diff
@@ -368,6 +368,15 @@ public void PauseKeyPressed()
         }
     }
 
+    public override void RecordPlayerReproduction()
+    {
+        if (!HasPlayer || Player.Get<Health>().Dead || PlayerIsEngulfed(Player))
```

Comment:
If the stage was in control when the player reproduction was recorded, this duplicate logic would not be needed.